From: Fabian Vogt <fvogt@suse.de>
Subject: Add completion for qdbus-qt5

qdbus is the Qt 4 variant, also complete for the Qt 5 version.

Index: bash-completion-2.11/completions/qdbus
===================================================================
--- bash-completion-2.11.orig/completions/qdbus
+++ bash-completion-2.11/completions/qdbus
@@ -9,6 +9,6 @@ _qdbus()
     COMPREPLY=($(compgen -W '$(command ${words[@]} 2>/dev/null | \
         command sed "s/(.*)//")' -- "$cur"))
 } &&
-    complete -F _qdbus qdbus dcop
+    complete -F _qdbus qdbus qdbus-qt5 dcop
 
 # ex: filetype=sh
Index: bash-completion-2.11/completions/Makefile.am
===================================================================
--- bash-completion-2.11.orig/completions/Makefile.am
+++ bash-completion-2.11/completions/Makefile.am
@@ -680,6 +680,7 @@ CLEANFILES = \
 	pyvenv-3.6 \
 	pyvenv-3.7 \
 	pyvenv-3.8 \
+    qdbus-qt5 \
 	qemu-kvm \
 	qemu-system-i386 \
 	qemu-system-x86_64 \
@@ -894,7 +895,7 @@ symlinks: $(DATA)
 	$(ss) pyvenv \
 		pyvenv-3.4 pyvenv-3.5 pyvenv-3.6 pyvenv-3.7 pyvenv-3.8
 	$(ss) qdbus \
-		dcop
+		dcop qdbus-qt5
 	$(ss) qemu \
 		qemu-kvm qemu-system-i386 qemu-system-x86_64
 	$(ss) quota \
